src/components/GameRoundNeighbors/GameRoundNeighbors.tsx

```diff
@@ -195,7 +187,8 @@ function GameRoundNeighbors({
           <div className="grid grid-cols-6 gap-1 text-center py-0.5">
             <div className="my-div-1">
               <span className="opacity-70">
-                {t("guessNoun")} {guesses.length + 1} / {maxAttempts}
+                {dataBank.tLang("guessNoun")} {guesses.length + 1} /{" "}
```

Comment:
prettier turned into uglier, perhaps undo these changes (it's on multiple lines)

Reply:
done